Apartment.rb

```diff
@@ -0,0 +1,3 @@
+class Apartment < ActiveRecord::Base
```

Comment:
The name of the class here is correct, in that it's Capitalized, but note that the name of the file should should `apartment.rb`. (note the lowercase). Our file names will almost always be lower_snake_case.